backend/Services/Services/RestaurantService.cs

```diff
@@ -104,7 +104,7 @@ public string Register(Credentials creds, RestaurantRegisterRequest restaurantRe
             // Validations
             if (_restaurantRepository.GetByMail(creds.Mail) != null)
             {
-                throw new Exception("There is already an account registered on this mail");
+                throw new ApplicationException("There is already an account registered on this mail");
```

Comment:
We should change this when custom exceptions are implemented.
[https://docs.microsoft.com/en-us/dotnet/api/system.applicationexception?view=net-5.0#remarks](url)

Reply:
Yeah I will change it when we have our custom exceptions, but for now I just needed a different exception.